### ISSUE TITLE
Application of quantum optimization

### ISSUE THREAD

**Issue by koreal6803** [qiskit-community/qiskit-hackathon-taiwan-20#22]
# Abstract
Our goal is to find the best solution to a problem (according to some criteria) from a set of possible solutions. Mostly, the optimization problem is formulated as a minimization problem, where one tries to minimize an error which depends on the solution: the optimal solution has the minimal error. We would like to apply quantum optimization problems to real world applications, such as finance portfolio optimization or vehicle location estimation.

# Topic Candidates
For now, our top choice is to perform localization for vehicle sensor network. We assume each vehicle is able to send signal to each other and estimation the signal strength. With the signal strength, vehicle can estimate the distance of the the sensor. Therefore, it can estimate its position by formulating the problem to quadratic functions and solved using quantum optimization.

# Members

 - @koreal6803
 - @mandy6803
 - @belmirochu
 - @chs14
 - @YanChengWeiTony
 - Qiskit Coach: @starktech23, @t-imamichi

# Deliverable
An application to help improve our life.

# GitHub repo
https://github.com/koreal6803/Application-of-quantum-optimization

**Comment by starktech23** [comment]
@mandy6803 @belmirochu @chs14 @YanChengWeiTony please drop a comment here to assign yourselves to this issue from the top-right corner, or I can assign you if you prefer that.

**Comment by starktech23** [comment]
@koreal6803 is this looking for a coach?

**Comment by mandy6803** [comment]
> @koreal6803 is this looking for a coach?

Yes, It will be nice to have you as our coach! Thank you for joining us!

**Comment by YanChengWeiTony** [comment]
Good

**Comment by belmirochu** [comment]
> @koreal6803 is this looking for a coach?

Yes, we are looking for a coach. thanks for your helping!!

**Comment by chs14** [comment]
Hi

**Comment by HuangJunye** [comment]
@t-imamichi Do you have the bandwidth to coach this team as well?

**Comment by mandy6803** [comment]
@starktech23 Thank you for being our coach. Please find the slack invitation link below to discuss our topic.
We will put our idea on it for you later to review. Look forward to your comment.
https://join.slack.com/t/qiskit/shared_invite/zt-hdktx2wg-j7OujbYiH5CGG1_m3Kk6Tw

**Comment by t-imamichi** [comment]
I see @starktech23 is already assigned as a coach. If they need any help, I can support them.

**Comment by starktech23** [comment]
@t-imamichi I didn't know I was assigned as coach, but I'll need your support as well if I am to coach 😆 sorry for the confusion.

**Comment by t-imamichi** [comment]
No problem!

**Comment by mandy6803** [comment]
@t-imamichi @starktech23 Thank you for both of your supports! Sorry for the confusion!

**Comment by starktech23** [comment]
@mandy6803 can you and the team please start by creating a Slack DM group to stay connected, and creating a GitHub repo with all of your team members as collaborators to work together? Add me as well so I can take a look whenever you need me to.

**Comment by starktech23** [comment]
Oh I see @koreal6803 already has a repo prepared. Everyone please join as contributors there and communicate via Slack.
